Audio/BnkCompiler/ProjectLoaderHelpers.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Audio.BnkCompiler.ObjectConfiguration.Warhammer3;
+using Audio.FileFormats.WWise.Hirc.Shared;
+using System;
+
+namespace Audio.BnkCompiler
+{
+    public class ProjectLoaderHelpers
+    {
+        private static readonly IVanillaObjectIds _VanillaObjectIds = new IdProvider();
+
+        public static int GenerateRandomNumber()
+        {
+            var rand = new Random();
```

Comment:
instread of random, would it be better if the user is allowed to ender a base number in the project file, that that is used and added to?